src/snap/invokeSnap.ts

```diff
@@ -0,0 +1,31 @@
+import { Page } from "puppeteer";
+import { SnapInstallationParamNames } from "../types";
+import { flaskOnly } from "./utils";
+
+export async function invokeSnap(
+  page: Page,
+  snapId: string,
+  method: string,
+  params: Record<SnapInstallationParamNames, any> = {}
+): Promise<Partial<any>> {
```

Comment:
avoid using `any` as params should be a JSON parable
The current type can be found on docks https://docs.metamask.io/guide/snaps-rpc-api.html#wallet-invokesnap